completions-sample/src/extension.ts

```diff
@@ -6,9 +6,9 @@
 
 import * as vscode from 'vscode';
 
-export function activate(context: vscode.ExtensionContext) {
+export function activate(extension: vscode.ExtensionContext) {
```

Comment:
> @jrieken cosmetic in the way of avoiding confusion & error prone behaviour, yes.

Given that, this rename isn't helpful because `extension` makes you think the object is the extension, but it's the context for that extension. You wanna be unambiguous you should use `extensionContext` and `completionContext`